docs/reference/ilm/policy-definitions.asciidoc

```diff
@@ -68,6 +73,8 @@ relative to when data was written last to a rolled over index.
 
 === Actions
 
+beta[]
```

Comment:
do we really need a beta[] tag per sub-header?

Reply:
Those break onto separate pages, so...yes. :-)